### ISSUE TITLE
reimporting cv make last-published date-time change

### ISSUE THREAD

**Issue by ldjebran** [SatelliteQE/robottelo#4620]
test affected 
tests.foreman.cli.test_import.TestImport.test_negative_reimport_cv
on 6.2 the test pass as the returned data does not contain last-published date-time field

how to consider this failure as bug or to refactor with removing last-published from assertion? 
 
```
        )
E               AssertionError: Lists differ: [[{u'[412 chars] 19:36:48', u'name': u'111-HsJl9JVFui', u'comp[1043 chars]1'}]] != [[{u'[412 chars] 19:37:37', u'name': u'111-HsJl9JVFui', u'comp[1043 chars]1'}]]
E               
E               First differing element 1:
E               [{u'last-published': u'2017/04/05 19:36:48', u'name': u'111-HsJl9JVFui', u'composite': u'false', u'repository-ids': u'266', u'label': u'111-HsJl9JVFui', u'content-view-id': u'490'}, {u'last-published': u'2017/04/05 19:35:57', u'name': u'Default Organization View', u'composite': u'false', u'repository-ids': u'', u'label': u'be87c29e-6560-4872-81e4-e70e35c3a2eb', u'content-view-id': u'486'}, {u'last-published': u'2017/04/05 19:36:47', u'name': u'HsJl9JVFui', u'composite': u'false', u'repository-ids': u'', u'label': u'HsJl9JVFui', u'content-view-id': u'489'}]
E               [{u'last-published': u'2017/04/05 19:37:37', u'name': u'111-HsJl9JVFui', u'composite': u'false', u'repository-ids': u'266', u'label': u'111-HsJl9JVFui', u'content-view-id': u'490'}, {u'last-published': u'2017/04/05 19:35:57', u'name': u'Default Organization View', u'composite': u'false', u'repository-ids': u'', u'label': u'be87c29e-6560-4872-81e4-e70e35c3a2eb', u'content-view-id': u'486'}, {u'last-published': u'2017/04/05 19:36:47', u'name': u'HsJl9JVFui', u'composite': u'false', u'repository-ids': u'', u'label': u'HsJl9JVFui', u'content-view-id': u'489'}]
E               
E               Diff is 2024 characters long. Set self.maxDiff to None to see it.
```

**Comment by omaciel** [comment]
@ldjebran @rplevka @lpramuk could we be more explicit about the attributes being used for the assertion so that we don't get tripped by differences in the time? I assume that the date and not time is more important for this test?

**Comment by lpramuk** [comment]
it's not a bug, CV gets simply republished upon next import run (re-import) thus time changes and 6.3 introduced this new field...

Anyway this will be resolved by the #5061 issue (Transitions subcommand removal)

**Comment by lpramuk** [comment]
Closing in favor of #5061
